### PR TITLE
added docker file

### PR DESCRIPTION
**Pull request by lokendarjangid**
#6 
- Docker file is been created

To build image - run 
''' docker build -t resullme . '''

To run container 
''' docker run -p 8501:8501 resullme '''

open the app in the browser
http://localhost:8501

### CODE REVIEW

**Comment by lokendarjangid** [comment]
list docker container runing command
''' docker ps '''

stop docker container
''' docker stop <container_id>

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+FROM python:3.9
+
+# set working directory
+WORKDIR /app
+
+# copy the dependencies file to the working directory
+COPY requirements.txt .
+
+# install dependencies
+RUN pip install -r requirements.txt
```
Comment (IvanIsCoding) [inline Dockerfile:14]: Or we add `apt install -y < packages.txt`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# light weight python streamlit app into a docker container 
+# https://hub.docker.com/r/continuumio/miniconda3
+
+# base image
+FROM python:3.9
```
Comment (IvanIsCoding) [inline Dockerfile:5]: Either we choose a base image already with Tex Live (recommended)

**Review by IvanIsCoding — COMMENTED** [review]
This runs Streamlit but does not render the PDF. You need to have some form of LaTex installed (e.g. Tex Live, TinyTex) to make the application work

**Comment by lokendarjangid** [comment]
fixed package installation error . after testing tex live base image i will update it here. This is working now

**Comment by IvanIsCoding** [comment]
> fixed package installation error . after testing tex live base image i will update it here. This is working now

Overall it LGTM, I’d say test it the LaTeX in the Render JSON section with https://github.com/IvanIsCoding/ResuLLMe/blob/main/.github/Turing.json just to make sure. I’ll test on my side later today when I have time

**Comment by lokendarjangid** [comment]
can you please provide api key for openai. my free tokens are expired.
mail : lokendarjangid1234@gmail.com

**Comment by IvanIsCoding** [comment]
> can you please provide api key for openai. my free tokens are expired. mail : [lokendarjangid1234@gmail.com](mailto:lokendarjangid1234@gmail.com)



> can you please provide api key for openai. my free tokens are expired. mail : [lokendarjangid1234@gmail.com](mailto:lokendarjangid1234@gmail.com)

You can test rendering without the OpenAI key by going to [http://localhost:8501/Render_JSON_Resume](http://localhost:8501/Render_JSON_Resume). Right now I am getting the error that pdflatex is not installed, I think I can fix it and then we can merge this

**Pull request merged**